bucket/v2rayn.json

```diff
@@ -16,6 +19,9 @@
         "if (!(Test-Path \"$persist_dir\\config.json\")) {",
         "    Copy-Item \"$(appdir v2ray $global)\\current\\config.json\" \"$dir\" | Out-Null",
         "}",
+        "if (!(Test-Path \"$persist_dir\\config.json\")) {",
```

Comment:
Does not make sense. The config from v2ray will be always overrided with the one from xray

Reply:
If there is already a config form v2ray, the second configuration will not be copied.